docker.local.cfg

```diff
@@ -0,0 +1,4 @@
+dspace.dir = /dspace
```

Comment:
As noted in today's meeting, let's move this to `[src]/src/main/docker/local.cfg`

Can we also add some comments at the top of this file to note that this is specific to Docker and used by the DockerFile (or something like that)?